### ISSUE TITLE
This configuration does not contain a TelemetryPingBuilder for focus-event

### ISSUE THREAD

**Issue by st3fan** [mozilla-mobile/telemetry-ios#33]
Firefox prints the following message:

```
This configuration does not contain a TelemetryPingBuilder for focus-event
```

Not sure if this is a telemetry library issue or a Firefox issue. Please move to Bugzilla if it is a Firefox issue.

**Comment by garvankeeley** [comment]
Good catch, not an error state (that message is correct for the way the lib is being used), we'll get this cleaned up when we get to the event part of the lib.

**Comment by justindarc** [comment]
This should now be fixed with the changes I landed in Issue #20: https://github.com/mozilla-mobile/telemetry-ios/commit/f9de22047b5afeac33e1c0ad4d5606c025942d22#diff-3e6b47f792970665f1ab46b28afde16d
